async/.await support (#9)
u32i64/rvk: README.md

<sub>`Cargo.toml`</sub>
```toml
[dependencies]
rvk = "0.20"
```

Now you can take a look at `rvk`'s [API documentation][docs.rs/rvk] to learn more about the available functions.

# Example

> To use this example, you will **also** need the [`serde_json`][crates.io/serde_json] crate to deserialize the API response.
> 
> If you want to use `tokio::main` proc macro, you need to install `tokio` crate.

Reviewer: ```suggestion
To use this example, you will **also** need the [`serde_json`][crates.io/serde_json] crate to deserialize the API response, and the [`tokio`](https://crates.io/tokio) crate for the `tokio::main` attribute proc macro.
```
